notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -840,6 +840,49 @@ facing a home invader [A sadistic janitor broke into a family's home, killed the
 terrorism [The viewer learns that Elijah was ultimately confined to an institution for the criminally insane for having perpetrated three acts of terrorism.]
 
 
+movie: The One (2001)
+=====================
+
+:: Title
+The One
+
+:: Description
+The One is a 2001 American science fiction action film directed by James Wong,
+written by Wong and Glen Morgan, and starring Jet Li, Delroy Lindo, Carla
+Gugino, and Jason Statham.
+
+Synopsis: A rogue agent travels to parallel realities in order for him to kill
+other versions of himself to become a mythical super-being known as "The One".
+
+:: Date
+2001-11-02
+
+:: References
+https://en.wikipedia.org/wiki/The_One_(2001_film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+multiverse [The story follows the former interdimensional police officer Gabriel Yulaw as he seeks to hunt down all variations of himself in alternate universes.]
+parallel universe [The story follows the former interdimensional police officer Gabriel Yulaw as he seeks to hunt down all variations of himself in alternate universes.]
+facing organized pursuit [Gabriel Yulaw was being pursued by MultiVerse Authority agents across multiple alternative realities.]
+law enforcement [Members of an agency to police the multiverse, called the MultiVerse Authority, were after one of their own who'd gone rogue and started hunting down all variations of himself in alternate universes. In one such alternate universe, the viewer is shown Gabe Law's experiences working as a police officer for the Los Angeles Police Department.]
+what if I met my double [The story followed the LAPD police officer Gabe Law as a version of him from another universe, Gabriel Yulaw, tried to hunt him down and kill him. In the end, Gabe used his martial arts skills to defeat Gabriel in an epic one-on-one fight.]
+speculative travel [A main novelty of the film is that MultiVerse Authority agents interdimensionally travel from one alternate universe to another by means of wormholes.]
```

Comment:
This is a can of wormholes. We have not been categorising wormhole traversal as speculative travel before. And if we start, surely it's a kind of space travel? Or need we include walking through magic doors and mirrors as speculative travel too?

Reply:
A big can of wormholes, indeed. I deleted it.